src/test/java/org/ays/auth/service/impl/AdminRegistrationCompleteServiceImplTest.java

```diff
@@ -373,15 +389,15 @@ void givenExistingEmailFromAdminRegisterRequest_whenAdminExist_thenThrowAysUserA
         );
 
         // Verify
-        Mockito.verify(adminRegistrationApplicationReadPort, Mockito.times(1))
-                .findById(Mockito.anyString());
-
         Mockito.verify(userReadPort, Mockito.times(1))
                 .existsByEmailAddress(Mockito.anyString());
 
         Mockito.verify(userReadPort, Mockito.never())
                 .existsByPhoneNumber(Mockito.any(AysPhoneNumber.class));
 
+        Mockito.verify(adminRegistrationApplicationReadPort, Mockito.never())
```

Comment:
Neden Mockito.times(1) iken Mockito.never() olarak değiştirildi?

Reply:
Artık telefon numarası ve email adres kontrolünden sonra çağırdığımız için buradaki testte bu metot çağrılmıyor bu yüzden bu şekilde düzenledim

Reply:
tamamdır, bunun gibi bir değişiklik daha var orada da o şekilde o zaman. Eline sağlık 🙌